app/frontend/javascript/lazy_member_images.js

```diff
@@ -0,0 +1,97 @@
+// GET a batch of thumbnails from the lazy images method in the public works controller. A typical path to GET would be:
+//      /works/WORK_FRIENDLIER_ID/lazy_member_images?start_index=100&images_per_page=100
+// and then insert them directly into the DOM.
+//
+// This is intended to speed up the perceived loading of books with over 500 pages.
+// See https://github.com/sciencehistory/scihist_digicoll/issues/905 for context.
+//
+// See also :
+//   app/components/work_image_show_component.rb
+//      (provides this script with friendlierID, startIndex, and imagesPerPage)
+//   app/controllers/works_controller.rb#lazy_member_images
+//      (provides this script with the images)
+//
+// Contains no JQuery code.
+
+class LazyMemberImages {
+
+  // this link won't be necessary; we'll just call the method automatically
+  constructor() {
+     document.querySelector(".lazy-member-images-link")?.addEventListener("click", this.#getMoreMemberImages.bind(this));
+  }
+
+  // Retrieve the images and insert them
+  #getMoreMemberImages(event) {
```

Comment:
I've never seen starting a js method with a `#` before.  Did you get this convention from somewhere, or does it mean something in JS I am not aware of?